### PR TITLE
HPCC-18601 Re-evaluate package maps widget in 7.0

### PR DESCRIPTION
**Pull request by miguelvazq**
Improving some of the code in PackageMapQueryWidget.js to take advantage
of the ECLWatch framework. U/I was not loading in 7.0.0 due
to some changes in dependencies - that has been fixed. Also, improve overall usability and
performance.

Signed-off by: Miguel Vazquez <miguel.vazquez@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [ ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by miguelvazq** [comment]
@GordonSmith Please review

File: esp/src/eclwatch/PackageMapQueryWidget.js [hunk 1]
```diff
@@ -87,42 +81,32 @@ define([
...
         },
 
-        startup: function (args) {
-            this.inherited(arguments);
-            this.refreshActionState();
-        },
+        // resize: function (args) {
+        //     this.inherited(arguments);
+        //     this.borderContainer.resize();
+        // },
```
Comment (GordonSmith) [inline esp/src/eclwatch/PackageMapQueryWidget.js]: Remove commented code.

**Review by GordonSmith — CHANGES_REQUESTED** [review]
Looks ok - just some commented code to remove.

**Comment by richardkchapman** [comment]
@miguelvazq

**Comment by miguelvazq** [comment]
@GordonSmith

**Comment by miguelvazq** [comment]
@GordonSmith Removed comments and rebased

File: esp/src/eclwatch/PackageMapQueryWidget.js [hunk 2]
```diff
@@ -431,108 +313,109 @@ define([
...
                     }
-                },
-                { name: this.i18n.Description, field: "Description", width: "20%" }
-            ]);
-            var objStore = ESPPackageProcess.CreatePackageMapQueryObjectStore();
-            this.packagesGrid.setStore(objStore);
+                }
+            }, this.id + "PackagesGrid");
 
             var context = this;
```
Comment (GordonSmith) [inline esp/src/eclwatch/PackageMapQueryWidget.js]: Already declared.

File: esp/src/eclwatch/PackageMapQueryWidget.js [hunk 3]
```diff
@@ -431,108 +313,109 @@ define([
...
-                { name: this.i18n.ProcessFilter, field: "Process", width: "15%" },
-                {
-                    name: this.i18n.Active,
-                    field: "Active",
-                    width: "10%",
-                    formatter: function (active) {
-                        if (active === true) {
-                            return "A";
+        initPackagesGrid: function () {
+            var context = this;
```
Comment (GordonSmith) [inline esp/src/eclwatch/PackageMapQueryWidget.js]: Already declared.

**Review by GordonSmith — CHANGES_REQUESTED** [review]
I suspect this will fail linting due to duplicate vars.

**Comment by miguelvazq** [comment]
@GordonSmith Updated.

**Comment by miguelvazq** [comment]
@GordonSmith I am not seeing where the smoketest would fail? Is this suspect? `Cant resolve hpcc/ESPUtil in /mnt/disk1/home/vamosax/smoketest/PR-10958/build/esp/src/tmp/eclwatch`

**Comment by miguelvazq** [comment]
@GordonSmith Should be good now.

**Review by GordonSmith — APPROVED** [review]
Looks good.

**Pull request merged**